content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -154,6 +154,12 @@ The kubelet has the following default hard eviction thresholds:
 * `imagefs.available<15%`
 * `nodefs.inodesFree<5%` (Linux nodes)
 
+These default values of hard eviction thresholds will only be set if none 
+of the parameter is changed. If you changed the value of any of the parameter, 
```

Comment:
Combining the suggestion by Chris above along with a slight amendment in grammar.
```suggestion
of the parameters is changed. If you changed the value of any parameter, 
```